tests/Http/CampaignTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Tests\Http;
+
+use Tests\TestCase;
+use Rogue\Models\Campaign;
+
+class CampaignTest extends Testcase
+{
+    /**
+     * Test that a POST request to /campaigns creates a new campaign.
+     *
+     * POST /campaigns
+     * @return void
+     */
+    public function testCreatingACampaign()
+    {
+        // Create a campaign.
+        $firstCampaignTitle = $this->faker->sentence;
+        $firstCampaignStartDate = $this->faker->date;
+        $firstCampaignEndDate = $this->faker->date;
+
+        $this->actingAsAdmin()->postJson('campaigns', [
+            'internal_title' => $firstCampaignTitle,
+            'start_date' => $firstCampaignStartDate,
+            'end_date' => $firstCampaignEndDate,
+        ]);
+
+        // Make sure the campaign is persisted.
+        $this->assertDatabaseHas('campaigns', [
+            'internal_title' => $firstCampaignTitle,
+            'start_date' => $firstCampaignStartDate,
+            'end_date' => $firstCampaignEndDate,
+        ]);
+
+        // Try to create a second campaign with the same title and make sure it doesn't duplicate.
+        $this->actingAsAdmin()->postJson('campaigns', [
+            'internal_title' => $firstCampaignTitle,
+            'start_date' => $this->faker->dateTime,
+            'end_date' => $this->faker->dateTime,
+        ]);
+
+        $response = $this->getJson('api/v3/campaigns');
+        $decodedResponse = $response->decodeResponseJson();
+
+        $this->assertEquals(1, $decodedResponse['meta']['pagination']['count']);
```

Comment:
@chloealee oooo I know you were having trouble hitting the endpoints but do these tests pass?

Reply:
@katiecrane they do!

Reply:
@chloealee that makes it seem like something is just up with postman to me!

Reply:
@katiecrane yes, agreed! :)